<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_AmcReset</class>
 <widget class="QDialog" name="Dial_AmcReset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amc Reset</string>
  </property>
  <widget class="QCheckBox" name="checkBox_Actived_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived_6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived_7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived_8">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived_9">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived_10">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived_11">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived_0">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Actived</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_11">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Rx_0">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>71</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Rx</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_Tx">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tx</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>241</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Asm</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Thor</string>
   </property>
   <widget class="QCheckBox" name="checkBox_ThorActived">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Actived</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Thor_Rx">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rx</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Thor_Tx">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tx</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_Daq">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Daq</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Gbe">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10 Gbe</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Tse">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Do">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Do it</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>421</width>
     <height>79</height>
    </rect>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>checkBox_Actived_1</zorder>
  <zorder>checkBox_Actived_2</zorder>
  <zorder>checkBox_Actived_3</zorder>
  <zorder>checkBox_Actived_4</zorder>
  <zorder>checkBox_Actived_5</zorder>
  <zorder>checkBox_Actived_6</zorder>
  <zorder>checkBox_Actived_7</zorder>
  <zorder>checkBox_Actived_8</zorder>
  <zorder>checkBox_Actived_9</zorder>
  <zorder>checkBox_Actived_10</zorder>
  <zorder>checkBox_Actived_11</zorder>
  <zorder>checkBox_Actived_0</zorder>
  <zorder>checkBox_Asm_Rx_1</zorder>
  <zorder>checkBox_Asm_Rx_2</zorder>
  <zorder>checkBox_Asm_Rx_3</zorder>
  <zorder>checkBox_Asm_Rx_4</zorder>
  <zorder>checkBox_Asm_Rx_5</zorder>
  <zorder>checkBox_Asm_Rx_6</zorder>
  <zorder>checkBox_Asm_Rx_7</zorder>
  <zorder>checkBox_Asm_Rx_8</zorder>
  <zorder>checkBox_Asm_Rx_9</zorder>
  <zorder>checkBox_Asm_Rx_10</zorder>
  <zorder>checkBox_Asm_Rx_11</zorder>
  <zorder>checkBox_Asm_Rx_0</zorder>
  <zorder>checkBox_Asm_Tx</zorder>
  <zorder>checkBox_Daq</zorder>
  <zorder>checkBox_Gbe</zorder>
  <zorder>checkBox_Tse</zorder>
  <zorder>pushButton_Do</zorder>
  <zorder>pushButton_Close</zorder>
  <zorder>textEdit_Msg</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
